src/nft/components/bag/profile/ListingModal.tsx

```diff
@@ -174,10 +168,12 @@ const ListingModal = () => {
     } else if (!paused) {
       setListingStatus(ListingStatus.FAILED)
     }
-    sendAnalyticsEvent(EventName.NFT_LISTING_COMPLETED, {
+    sendAnalyticsEvent(EventName.NFT_LISTING_SIGNED, {
       signatures_requested: listings.length,
       signatures_approved: listings.filter((asset) => asset.status === ListingStatus.APPROVED),
-      ...approvalEventProperties,
+      list_quantity: listings.length,
+      usd_value: ethPriceInUSD * totalEthListingValue,
```

Comment:
If this event is supposed to be for individual listings it should be part of the function `signListingRow` right now this only triggers once after all the signature requests have been complete (approved, rejected, failed, etc)

Otherwise what is list_quantity supposed to represent here and how is it different than `signatures_requested` and `signatures_approved`?

Reply:
you actually should probably move this to `signListing` fn in `src/nft/utils/listNfts.ts` which is per listing and will give the tx hash context requested in the ticket